### PR TITLE
Testing/log

### PR DESCRIPTION
**Pull request by jim-parry**
Flesh out Log module testing, with near 100% code coverage.
Fixes bug in ChromeLoggerHandler re error level conversion.

### CODE REVIEW

File: log-2018-05-22.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php defined('BASEPATH') || exit('No direct script access allowed'); ?>
```
Comment (lonnieezell) [inline log-2018-05-22.php]: Don't think you meant to include this one...

Comment (jim-parry) [reply log-2018-05-22.php]: Nope - that should have been git ignored. Will fix

**Comment by jim-parry** [comment]
@lonnieezell Merged your testing changes here, & sorted out the namespaces.
Travis-ci runs, though no code coverage, but this still croaks trying to run locally - the same as the comment on your merged PR.
I am not comfortable merging this until I resolve phpunit locally.

**Comment by lonnieezell** [comment]
@jim-parry Is this good to go now?

**Comment by jim-parry** [comment]
It is indeed good to go. I didn't want to merge it directly, in case you had any concerns.

**Pull request merged**